apps/web/src/components/submission/validation/index.ts

```diff
@@ -1,8 +1,8 @@
-import { ContactInformationDTO, PayloadDTO } from '@ehpr/common';
+import { ContactInformationDTO, FormPayload } from '@ehpr/common';
```

Comment:
I really liked `PayloadDTO` over `FormPayload` to keep the naming convention consistent.